pkg/storage/image.go

```diff
@@ -553,9 +554,17 @@ func (svc *imageService) ResolveNames(imageName string) ([]string, error) {
 		}
 		return nil, err
 	}
+
 	domain, remainder := splitDockerDomain(imageName)
+	// this means the image is already fully qualified
```

Comment:
(Absolutely non-blocking: The old location of this comment inside the `if` was a bit clearer.)

Reply:
done